docs/en_US/Tutorial/ExperimentConfig.md

```diff
@@ -1,8 +1,8 @@
 # Experiment config reference
 
-A config file is needed when create an experiment, the path of the config file is provide to nnictl.
-The config file is written in YAML format, and need to be written correctly.
-This document describes the rule to write config file, and will provide some examples and templates.
+A config file is needed when creating an experiment. The path of the config file is provide to `nnictl`.
```

Comment:
Thanks, @daniel347x . Could you kindly change 'provide' to 'provided'?

Reply:
All set, thanks.